### PR TITLE
Update article.md

### PR DESCRIPTION
**Pull request by hrodward**
Grammar

### CODE REVIEW

File: 1-js/08-prototypes/01-prototype-inheritance/article.md [hunk 1]
```diff
@@ -246,13 +246,13 @@ The resulting picture:
 
 ![](proto-animal-rabbit-walk-3.svg)
 
-If we had other objects like `bird`, `snake` etc inheriting from `animal`, they would also gain access to methods of `animal`. But `this` in each method call would be the corresponding object, evaluated at the call-time (before dot), not `animal`. So when we write data into `this`, it is stored into these objects.
+If we had other objects like `bird`, `snake` etc. inheriting from `animal`, they would also gain access to methods of `animal`. But `this` in each method call would be the corresponding object, evaluated at the call-time (before dot), not `animal`. So when we write data into `this`, it is stored into these objects.
```
Comment (lex111) [inline 1-js/08-prototypes/01-prototype-inheritance/article.md]: @paroche don't we need a comma here?

```suggestion
If we had other objects like `bird`, `snake` etc., inheriting from `animal`, they would also gain access to methods of `animal`. But `this` in each method call would be the corresponding object, evaluated at the call-time (before dot), not `animal`. So when we write data into `this`, it is stored into these objects.
```

Comment (paroche) [reply 1-js/08-prototypes/01-prototype-inheritance/article.md]: The place where I would add a comma is between "'snake'"  and "etc.". I think a comma after "etc." would only be appropriate if there were also a comma between "objects" and "like". So:

"If we had other objects like `bird`, `snake`, etc. inheriting from 'animal'..."

or

"If we had other objects, like 'bird', 'snake', etc., inheriting from 'animal'..."

Comment (lex111) [reply 1-js/08-prototypes/01-prototype-inheritance/article.md]: >  I think a comma after "etc." would only be appropriate

This is what I wanted to know and reflected in my suggestion.

@hrodward please accept the suggestion above.

**Comment by hrodward** [comment]
I clicked on “resolve discussion”. Is that enough?

**Comment by lex111** [comment]
@hrodward please click on "Commit suggestion" button.

**Comment by hrodward** [comment]
Done!

File: 1-js/08-prototypes/01-prototype-inheritance/article.md (hunk 1 above)
Comment (paroche) [reply 1-js/08-prototypes/01-prototype-inheritance/article.md]: I entered a reply here a few hours ago, but I can no longer find it. To paraphrase what I said then, I don't agree with this change. I would suggest a comma **before** "etc.", but would only recommend a comma **after** "etc." IFF there were also a comma before "like" (after "objects"), as in my examples above.

Comment (hrodward) [reply 1-js/08-prototypes/01-prototype-inheritance/article.md]: I agree with @paroche. My preferred option would be the comma before "etc."

Comment (lex111) [reply 1-js/08-prototypes/01-prototype-inheritance/article.md]: Oops, fine, let's make it.

Comment (paroche) [reply 1-js/08-prototypes/01-prototype-inheritance/article.md]: Done. Went w/ 2nd (3 comma) option.

**Pull request merged**